packages/edit-site/src/components/page-pages/index.js

```diff
@@ -307,11 +330,14 @@ export default function PagePages() {
 					paginationInfo={ paginationInfo }
 					fields={ fields }
 					actions={ actions }
+					bulkActions={ bulkActions }
```

Comment:
Have you explored if we can have a single `actions` API that also support an action to be a bulk one?

Reply:
An exploration at this point where we introduce a new API would be useful I think. I'm not opposed to this approach, but we need to explore if we can combine them. It could lead to a more elegant API or a more complex one 😅

Reply:
I feel like we should only have one API personally. An action is just a bulk action that applies to a single item. I believe we can have flags about whether an action can apply to multiple items or not.